.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
         
     strategy:
       matrix:
-        go-version: [1.13.x]
+        go-version: [1.17.9]
```

Comment:
We generally specify 1.13.x, 1.17.x. 
Do we need to keep it specific to version 1.17.9?